packages/bento-design-system/src/Table/Table.tsx

```diff
@@ -236,6 +241,29 @@ export function Table<C extends ReadonlyArray<ColumnType<string, {}, any>>>({
     }
   }, [data.length, stickyLeftColumnsIds]);
 
+  const tableContainerRef = useRef<HTMLDivElement>(null);
+
+  const virtualizeRows =
+    typeof virtualizeRowsConfig === "boolean" ? virtualizeRowsConfig : virtualizeRowsConfig != null;
+  const estimateSize =
+    typeof virtualizeRowsConfig === "boolean"
+      ? () => 52
```

Comment:
not sure how to pick a proper default value here. In practice, testing it, it doesn't seem to change much